.github/workflows/iroha2-dev-pr.yml

```diff
@@ -44,23 +45,39 @@ jobs:
         run: mold --run cargo build --target wasm32-unknown-unknown --quiet
 
   tests_with_coverage:
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, Linux]
     container:
       image: 7272721/i2-ci:latest
+    strategy:
+      matrix:
+        feature_flag: [all-features, no-default-features]
     steps:
       - uses: actions/checkout@v3
-      # TODO Remove these steps #2165
+      # TODO Remove this step #2165
```

Comment:
This line is culpable for all our troubles with CI this week.

Reply:
1. Now that I don't know what `7272721/i2-ci` is, I don't intend to address #2165 in this PR
2. If you are referring to [this](https://github.com/hyperledger/iroha/runs/7836879507?check_suite_focus=true), [this](https://github.com/hyperledger/iroha/runs/7836323458?check_suite_focus=true) and [this](https://github.com/hyperledger/iroha/runs/7805599573?check_suite_focus=true), one of the authors can copy & paste and that's it.  I've also done

Reply:
> one of the authors can copy & paste and that's it.

No. Conversely, the version is degrading. The problem is that Rust `1.63` is expected and actually is `1.62`